docs-content/general/6_product-features/2_error-monitoring/enhancing-errors-with-github.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Enhancing Errors with GitHub
+slug: enhancing-errors-with-github
+createdAt: 2023-09-07T16:07:50.273Z
+updatedAt: 2023-09-07T16:07:50.273Z
+---
+
+Highlight has to capability to enhance your backend errors using GitHub. On the frontend, errors are enhanced using sourcemaps uploaded to Highlight. Some backend
```

Comment:
```suggestion
Highlight has the capability to enhance your backend errors using GitHub (errors on the frontend are enhanced using sourcemaps). Some backend
```

Is there a sourcemap doc we can link here in case someone lands on the wrong doc?

Reply:
Yep! Will add that link in here